_includes/newcomers/newcomers.html

```diff
@@ -0,0 +1,99 @@
+<h3>Welcome Newcomers!</h3>
+
+<p>
+    Whether you are a first-time contributor or an open-source veteran, you are welcome to contribute to and actively
+    engage in the development of projects at Layer5.
+</p>
+<p>
+    Here are some instructions to get you started and if you haven't joined yet, join the <a
+        href="http://slack.layer5.io" target="_blank">Slack channel</a> to collaborate with the
+    community.
+</p>
+<p>
+    Also, you can check the quick-links below for jumping straight into things.
+</p>
+<table style="margin:0px; padding:0px;width:100%; table-layout: fixed;">
+    <th>Information</th>
+    <th>Guides</th>
+    <th>Issues up for grabs</th>
+    <tr>
+        <td><a href="https://bit.ly/2SbrRhe" target="_blank">Calender</a> | <a href="http://slack.layer5.io"
+                target="_blank">Slack</a> | <a href="https://github.com/topics/meshery" target="_blank">GitHub</a> | <a
+                href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1"
+                target="_blank">YouTube</a> | <a href="https://drive.google.com/drive/folders/0ABH8aabN4WAKUk9PVA"
+                target="_blank">Drive</a></td>
+        <td><a href="https://docs.google.com/document/d/1tpg2sLxirozNt3Ofr3GdM002f9rExp74EqrsGZBU710/edit?usp=sharing"
+                target="_blank">First-timers' Welcome Guide</a></td>
+        <td><a href="https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22help+wanted%22+"
+                target="_blank">help-wanted</a></td>
+    </tr>
+    <tr>
+        <td> </td>
+        <td><a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit"
+                target="_blank">Welcome Newcomers!</a></td>
+        <td><a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3Ahacktoberfest"
+                target="_blank">Hacktoberfest</a></td>
```

Comment:
Hacktoberfest category can be removed. As most of those issues would have been addressed, so no point of adding them under issues for grabs.